pkg/build/vendor/github.com/docker/docker/pkg/term/windows/ansi_reader.go

```diff
@@ -142,14 +142,14 @@ func readInputEvents(fd uintptr, maxBytes int) ([]winterm.INPUT_RECORD, error) {
 
 // KeyEvent Translation Helpers
 
-var arrowKeyMapPrefix = map[uint16]string{
+var arrowKeyMapPrefix = map[winterm.WORD]string{
```

Comment:
This is an UPSTREAM commit that mirrors the patch we carry on the main vendor dir.